### PR TITLE
Added description and example to imageFormats()

### PR DESCRIPTION
**Pull request by SebastianZ**

### CODE REVIEW

File: data/en/imageformats.json [hunk 1]
```diff
@@ -4,11 +4,19 @@
...
+	"description": "Returns all supported image formats that can be read and written.",
 	"params": [],
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/imageformats.html"}
 	},
-	"examples": [],
+	"examples": [
+    {
+      "title": "Simple example",
+      "description": "Dumps the structure returned by the function",
```
Comment (shaedrich) [inline data/en/imageformats.json]: Could you explain what happens here better?

Comment (SebastianZ) [reply data/en/imageformats.json]: I've improved the description a in https://github.com/foundeo/cfdocs/pull/870/commits/0e8ea960f31d382c553cff4df1cbd8e1c010140d. Hope that makes it a bit clearer what the example does.

File: data/en/imageformats.json [hunk 2]
```diff
@@ -4,11 +4,19 @@
 	"syntax": "imageFormats()",
 	"returns": "struct",
 	"related": [],
-	"description": "",
+	"description": "Returns all supported image formats that can be read and written.",
```
Comment (shaedrich) [inline data/en/imageformats.json]: Could you assign `encoder`/`decoder` to read/write because this are the keys, the output structure uses?

Comment (SebastianZ) [reply data/en/imageformats.json]: Done in https://github.com/foundeo/cfdocs/pull/870/commits/09111c0648dea63fb08b6cbe127d90ebf8417ec5.

**Comment by pfreitag** [comment]
Thanks again @SebastianZ

**Pull request merged**